src/cryptography/hazmat/backends/openssl/aead/__init__.py

```diff
@@ -0,0 +1,132 @@
+# This file is dual licensed under the terms of the Apache License, Version
```

Comment:
No need for an `aead` subdirectory, these can just be `aead.py` and then the openssl/boringssl variants.

Reply:
Sounds good, will collapse into a single file.

Any thoughts on naming here? The "boring" impl now covers both BoringSSL and LibreSSL.

Reply:
evp_aead?

Reply:
Done; I've also prefixed the OpenSSL internals with `_openssl`, to keep things unambiguous.